contracts/rollover/CrossCurrencyRollover.sol

```diff
@@ -0,0 +1,437 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.18;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
+
+import "@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
+
+import "../origination/OriginationController.sol";
+
+import "../interfaces/ICrossCurrencyRollover.sol";
+import "../interfaces/ILoanCore.sol";
+import "../interfaces/IRepaymentController.sol";
+import "../libraries/LoanLibrary.sol";
+import "../libraries/InterestCalculator.sol";
+
+import {
+    CCR_UnknownBorrower,
```

Comment:
Can you double check these errors and remove the ones not being used. First 4 dont look to be used anymore.